backend/python/app/services/implementations/story_service.py

```diff
@@ -219,54 +219,96 @@ def assign_user_as_reviewer(self, user, story_translation):
             self.logger.error("User can't be assigned as a reviewer")
             raise Exception("User can't be assigned as a reviewer")
 
+    # Deprecated: function is not currently in use (story translation stage logic has not been tested)
     def update_story_translation_content(self, story_translation_content):
-        try:
-            story_translation = StoryTranslationContent.query.filter_by(
-                id=story_translation_content.id
-            ).first()
+        story_translation = StoryTranslationContent.query.filter_by(
```

Comment:
ditto comment

Reply:
Can leave this one because it's in the deprecated function?